docs-kits/kits/Digital Twin Kit/page_software-operation-view.md

```diff
@@ -18,13 +18,25 @@ Digital Twins the Catena-X-way. This infrastructure empowers Data Consumers to c
+|-----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------|
+| Discovery Finder      | A microservice resolving type of identifiers against a set of BPN-Discovery Servers.                                                              | [Tractus-X Discovery Finder](https://github.com/eclipse-tractusx/sldt-discovery-finder)                                                                                                                         | CX - 0053                                                   |
+| BPN Discovery         | A microservice resolving a particular assetId against the registered BPN of its owner.                                                            | [Tractus-X BPN Discovery](https://github.com/eclipse-tractusx/sldt-bpn-discovery)                                                                                                                               | CX - 0053                                                   |
+| EDC Discovery         | A microservice that resolves a BPN against an EDC endpoint.                                                                                       | [Tractus-X Portal including EDC Discovery API](https://github.com/eclipse-tractusx/portal-backend)                                                                                                              | CX - 0001                                                   |
+| Digital Twin Registry | A registry for digital twins, each digital twin providing the endpoints of its submodels. <br/> The registry is adhering to the AAS Registry API. | [Tractus-X Digital Twin Registry](https://github.com/eclipse-tractusx/sldt-digital-twin-registry)                                                                                                               | CX - 0002                                                   |
+| Submodel Server       | The data source adhering to a subset of the Submodel API as defined in AAS Part-2 3.0.                                                            | [FA³ST-Framework](https://github.com/FraunhoferIOSB/FAAAST-Service), <br/> [Eclipse Basyx](https://github.com/eclipse-basyx/basyx-java-sdk), <br/> [AASX Server](https://github.com/admin-shell-io/aasx-server) | CX - 0002                                                   |
 
-![High Level Architecture of an App leveraging the Digital Twin Kit](assets/img/DTKIT_high_level_arch.svg)
\ No newline at end of file
+
+![High Level Architecture of an App leveraging the Digital Twin Kit](assets/img/DTKIT_high_level_arch.svg)
```

Comment:
Is this a self created image especially for the KIT please follow: https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-07

Reply:
TRG-7 says "if a figure in docs is self-created, the ##NOTICE paragraph at the end of the file is sufficient." The Adoption View already has one.